.gitignore

```diff
@@ -6,3 +6,4 @@ dist/*.js
 dist/*.css
 dist/img/*.svg
 data/data.js
+js/lib/id/*.js
```

Comment:
To preserve the ability for people to check out the project and use it without a build step, I think we want to keep `js/lib/id/index.js` checked in for now. Even though it is a generated file, it's used by `index.html` in project root.